### PR TITLE
📝 Fixed some grammar and punctuation

### PR DESCRIPTION
**Pull request by thedipankarroy**
Fixed some grammar and punctuation

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 [![Downloads](https://img.shields.io/npm/dt/zustand.svg?style=flat&colorA=000000&colorB=000000)](https://www.npmjs.com/package/zustand)
 [![Discord Shield](https://img.shields.io/discord/740090768164651008?style=flat&colorA=000000&colorB=000000&label=discord&logo=discord&logoColor=ffffff)](https://discord.gg/poimandres)
 
-A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks, isn't boilerplatey or opinionated.
+A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks that isn't boilerplatey or opinionated.
```
Comment (sewera) [inline readme.md:11]: It's a little bit different semantically now. Previously, not being boilerplatey referred to Zustand. Now it's about API.

But I think this version is also alright.

File: readme.md [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
 
-Don't disregard it because it's cute. It has quite the claws, lots of time was spent dealing with common pitfalls, like the dreaded [zombie child problem](https://react-redux.js.org/api/hooks#stale-props-and-zombie-children), [react concurrency](https://github.com/bvaughn/rfcs/blob/useMutableSource/text/0000-use-mutable-source.md), and [context loss](https://github.com/facebook/react/issues/13332) between mixed renderers. It may be the one state-manager in the React space that gets all of these right.
+Don't disregard it because it's cute. It has quite the claws, lots of time was spent dealing with common pitfalls, like the dreaded [zombie child problem](https://react-redux.js.org/api/hooks#stale-props-and-zombie-children), [react concurrency](https://github.com/bvaughn/rfcs/blob/useMutableSource/text/0000-use-mutable-source.md), and [context loss](https://github.com/facebook/react/issues/13332) between mixed renderers. It may be the one state-manager in the React space that gets all of these rights.
```
Comment (sewera) [inline readme.md:13]:
```suggestion
Don't disregard it because it's cute. It has quite the claws, lots of time was spent dealing with common pitfalls, like the dreaded [zombie child problem](https://react-redux.js.org/api/hooks#stale-props-and-zombie-children), [react concurrency](https://github.com/bvaughn/rfcs/blob/useMutableSource/text/0000-use-mutable-source.md), and [context loss](https://github.com/facebook/react/issues/13332) between mixed renderers. It may be the one state-manager in the React space that gets all of these right.
```

Gets all of these (as in solutions to common pitfalls) **right**. "Right" as an adjective, not a noun.

Please revert this change.

**Review by sewera — CHANGES_REQUESTED** [review]
The title of this PR is not correct. You fixed grammar and punctuation, not typos.

**Comment by thedipankarroy** [comment]
Thanks for Reviewing the PR.
Apologies for not naming the issue correctly and changed the file as you suggested.

File: readme.md (hunk 1 above)
Comment (thedipankarroy) [reply readme.md:11]: Sure then!

**Comment by thedipankarroy** [comment]
Thanks for Reviewing the PR.
And apologies for not naming the PR correctly and also changed the file as you suggested.

**Comment by sewera** [comment]
No worries, that's what a review is for

**Review by sewera — APPROVED** [review]

**Comment by thedipankarroy** [comment]
Is there anything else for me to do?

File: readme.md (hunk 1 above)
Comment (dai-shi) [reply readme.md:11]: I don't know but the original one is more intentional?

**Comment by sewera** [comment]
No, that's why I gave you an approval, should appear like this:

image

**Review by sewera — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (dai-shi) [reply readme.md:11]: > I don't know but the original one shows the intention more?

Let me repost it. Is there a grammar fix to preserve the original meaning?

Comment (dai-shi) [reply readme.md:11]: > I don't know but the original one shows the intention more?

Let me repost it. Is there a grammar fix to preserve the original meaning?

**Review by dai-shi — COMMENTED** [review]
Can you check please?

File: readme.md (hunk 1 above)
Comment (thedipankarroy) [reply readme.md:11]: Here are some examples, let's see which works best in this scenario.

- It has a comfortable API based on hooks that isn't boilerplatey or opinionated.
- Has a nice way of working that uses hooks and isn't too standard or opinionated.
-  Has a slick API built on hooks that's not stale or snarky.
- Has a nice API that's not too formal or opinionated.

Comment (dai-shi) [reply readme.md:11]: I think what's unopinionated is Zustand itself, not the API. @sewera Please help on this.

Comment (sewera) [reply readme.md:11]: Technically speaking, the only way of interacting with Zustand is through its API. That's why I approved that version as well. With that said, if the intention was specifically to mark Zustand as not opinionated or boilerplatey, let's go with the previous version.

Comment (sewera) [reply readme.md:11]: Now when I think of it, the previous version is probably better.

Comment (dai-shi) [reply readme.md:11]: Yeah, I think Zustand is an unopinionated library and its API allows less boilerplate.
Let's revert the change.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Let's revert the first change.

**Review by sewera — APPROVED** [review]

**Review by dai-shi — APPROVED** [review]
Thanks for your contribution!
Please feel free to look into `docs` and fix some grammar.
